src/sign-typed-data.ts

```diff
@@ -173,6 +173,7 @@ function encodeField(
   version: SignTypedDataVersion.V3 | SignTypedDataVersion.V4,
 ): [type: string, value: any] {
   validateVersion(version, [SignTypedDataVersion.V3, SignTypedDataVersion.V4]);
+  console.log('ENCODING', types, name, type, value);
```

Comment:
This console.log probably should be removed before merging. 👍

Reply:
whoops, forgot about that one :)